test/e-state-county/audit.bash

```diff
@@ -17,17 +17,15 @@ mkdir -p $LOGDIR
     ./main.py reset
 
     echo -e "\nStart 'Define audit' step and upload county audit data\n"
-    ./main.py dos_init
+    ./main.py dos_init -s 12341234123412341234 -r 0.05
```

Comment:
Can you add a comment (perhaps at the top of the file) on the expected results given this particular seed and risk limit? E.g. perhaps these changes are necessary for this to be a good test case for this PR (assuming that is true)?